src/engine/Preprocessor.cpp

```diff
@@ -560,14 +576,24 @@ void Preprocessor::collectFixedValues()
             }
             else
             {
-                _fixedVariables[i] = _preprocessed.getLowerBound( i );
+                if ( FloatUtils::isFinite( _preprocessed.getLowerBound( i ) ) )
+                    _fixedVariables[i] = _preprocessed.getLowerBound( i );
+                else if ( FloatUtils::isFinite( _preprocessed.getUpperBound( i ) ) )
+                    _fixedVariables[i] = _preprocessed.getUpperBound( i );
+                else
+                    _fixedVariables[i] = 0;
             }
+
+            _preprocessed.setLowerBound( i, _fixedVariables[i] );
+            _preprocessed.setUpperBound( i, _fixedVariables[i] );
         }
-	}
+    }
 }
 
 void Preprocessor::eliminateVariables()
 {
+    printf( "PP: eliminateVariables starting!\n" );
```

Comment:
Undo change - printfs shouldn't be merged to master, debug printing should be made using the LOG macros that can be turned on/off